index.js

```diff
@@ -47,6 +68,9 @@ export default {
     contentType?: string,
     streamDuration?: number,
     playPosition?: number,
+    isLive?: Boolean,
```

Comment:
This refers to the class `Boolean`. `boolean` is the right type to use here.

I was wondering how I can add a MediaTrack to the item that I'm casting. Do I need to pass the url to the VTT files in customData?

Reply:
Correcting to `boolean`, thanks.

As for attaching VTT files, we would need to alter `GoogleCastModule#castMedia` to accept extra options. Out of those we would need to build a MediaTrack instance to add to the MediaInfo object as per https://developers.google.com/cast/docs/android_sender/media_tracks

My change simply enables a specific (or default's) language subtitles that are already defined in the stream itself (or added prior in castMedia once that is in place).